pom.xml

```diff
@@ -43,7 +43,8 @@
 
         <air.java.version>17</air.java.version>
         <air.check.skip-enforcer>true</air.check.skip-enforcer>
-        <air.check.skip-dependency>true</air.check.skip-dependency>
+        <air.check.skip-dependency>false</air.check.skip-dependency>
+        <air.check.fail-dependency>false</air.check.fail-dependency>
```

Comment:
leave this out as well .. we are fixing all this in separate work

Reply:
updated both by reverting the changes